index.html

```diff
@@ -1,299 +1,417 @@
+        <div class="deskPosRelative">
+          <li class="liPaddingRight">Hello</li>
+          <img class="deskPosAbsolute" src="resources/underlinestick.svg" alt="Deco underlining stick">
+        </div>
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
+      </ul>
+  
+      <main>
+        <section class="mainPadding deskPosRelative">
+          <div class="deskPresentation">
+            <h1 class="playfairFont">José Fernández-Caminero</h1>
+  
+            <p class="paragraphStyle">I'm a Full-Stack engineer. I love coding beautiful, effecient, complex, difficult
+              websites.</p>
+  
+            <button class="buttonDeco mainButtonWidthDesk buttonTransition">Start collaboration</button>
+          </div>
+  
+          <div class="deskPresentationDeco">
+            <div class="dottedImage"></div>
+            <div class="bigDeco"></div>
+          </div>
+        </section>
+  
+        <section class="middleDeco"></section>
+  
+        <a id="Portfolio"></a>
+  
+        <section class="deskPosRelative">
+          <h2 class="playfairFont projectsTag deskPosAbsolute">Projects</h2>
+  
+          <img class="deskStuff deskPosAbsolute positionDeco1" src="resources/image-geometry_2.svg"
+            alt="Geometrical decoration">
+          <img class="deskStuff deskPosAbsolute positionDeco2" src="resources/Group83.svg"
+            alt="Squared decoration with many colors">
+          <img class="deskStuff deskPosAbsolute positionDeco3" src="resources/image-geometry_5.svg"
+            alt="Geometrical decoration, squares, and circles">
+          <img class="deskStuff deskPosAbsolute positionDeco4" src="resources/redSquare.svg" alt="Red square">
+          <img class="deskStuff deskPosAbsolute positionDeco5" src="resources/image-geometry_4.svg"
+            alt="Yellow geometrical decoration">
+          <img class="deskStuff deskPosAbsolute positionDeco6" src="resources/image_dots.png" alt="Dots">
+  
+          <div class="dgrid deskPosRelative">
+            <section class="flex-column firstCardDesk">
+              <img id="projectImage1" class="fullWidth bigImage" src="resources/Rectangle 21.svg" alt="Laptop, tangerine, and other objects">
+              <div class="darkBlueBackground projectsPadding flex-column firstCardBlueBackground">
+                <h2 class="projectnameStyle deskFont1" id="ProjectName1">Project name goes here</h2>
+  
+                <ul class="greenul">
+                  <li class="greenli" id="1tech1">HTML/CSS</li>
+                  <li class="greenli" id="1tech2">Ruby on Rails</li>
+                  <li class="greenli" id="1tech3">JavaScript</li>
+                </ul>
+  
+                <button id="projectButton1" class="buttonDeco2 buttonDecoDesk">See this project
+                  <img src="resources/Union.png" alt="Cool (White) Arrow Pointing Right">
+                  <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
```

Comment:
- [ ] As I already suggested in the other comment, the requirement for this milestone involves you removing these project lists from the HTML and generating them dynamically with JavaScript, and then you can append them to the page. :smile: :+1: 

> You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.

- Please consider looking into this. :small_airplane: